docs/contributing.md

```diff
@@ -178,8 +183,13 @@ Then you can translate it in `locale/eo/LC_MESSAGES/opsdroid.po`, then compile i
 Opsdroid is an open source chatbot framework. It is designed to be extendable, scalable and simple. It comes with a few official modules that can be found in the [Opsdroid  GitHub account](https://github.com/opsdroid).
 
  If you love a particular platform and wish to use opsdroid with it or if you want opsdroid to interact with something in a certain way you can create your own modules and extend the functionality of opsdroid.
+<<<<<<< HEAD
```

Comment:
This is the start of the conflict. You should delete the one that is outdated and go with the latest one.